### PR TITLE
Add help window and remove pre/post-processing checkboxes

### PR DESCRIPTION
**Pull request by ItsSimko**
## Summary by Sourcery

Enhance the settings UI by removing redundant pre-processing and post-processing checkboxes and introduce a help button to access detailed settings documentation.

Documentation:
- Add a comprehensive settings documentation in markdown format.

### CODE REVIEW

File: src/FreeScribe.client/markdown/help/settings.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+- **Use best_of**
+  - Description: Enable best-of sampling
+  - Default: `false`
+  - Type: boolean
+- **best_of**
+  - Description: Number of completions to generate and select from
+  - Default: `2`
+  - Type: integer
+- **Real Time Audio Length**
+  - Description: Length of audio segments for real-time processing (seconds)
```
[the remark this answers is not in the document]
Comment (ItsSimko) [reply src/FreeScribe.client/markdown/help/settings.md:127]: No its in adv for a reason

File: src/FreeScribe.client/markdown/help/settings.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+- **top_p**
+  - Description: Top-P (nucleus) sampling parameter
+  - Default: `0.4`
+  - Type: float
+- **typical**
+  - Description: Typical sampling parameter
+  - Default: `0.19`
+  - Type: float
+- **sampler_order**
+  - Description: Order of sampling methods to apply
```
[the remark this answers is not in the document]
Comment (ItsSimko) [reply src/FreeScribe.client/markdown/help/settings.md:103]: I think its OKAY since its and advanced setting

**Comment by yingbull** [comment]
What's the motivation here?

File: src/FreeScribe.client/UI/SettingsWindow.py [hunk 3]
```diff
@@ -123,8 +123,6 @@ def __init__(self):
             "Use best_of",
             "best_of",
             "Real Time Audio Length",
-            "Use Pre-Processing",
-            "Use Post-Processing",
```
Comment (ItsSimko) [inline src/FreeScribe.client/UI/SettingsWindow.py:127]: Instead of making them auto get listed, we manually did them above each textbox so this is OK

**Review by yingbull — APPROVED** [review]

**Pull request merged**